index.html

```diff
@@ -6,28 +6,30 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
-    <title>Document</title>
+    <title>Portfolio Kazaneza</title>
 </head>
 <body>
 
-    
+    <header>
         <nav class="nav">
             <label class="logo">KAZANEZA GENTIL</label>
             <a href="#">
                 <img class="navImage" src="./Images/ic_menu.svg" alt="Navigation menu">
             </a>
         </nav>
+    </header>   
+        
     
-        <header class="container">
+        <section class="container">
             <div class="headline"> Hey there. <br/> I'm Kazaneza</div>
             <div class="title">I’am a Software Developer</div>
             <div class="introduction"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, on’t hestiate to contact me.</div>
```

Comment:
- [ ] As requested earlier please use `<h1>` tag for your main title
  - If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. 
    [Check this link](https://www.reliablesoft.net/h1-tag/) for more info

- [ ] please make sure, The subtitle uses a header tag `hx (examples: <h2>,<h3>, etc...)`.
      - This way will be semantically correct and the screen readers and other technologies will thank you.

- [ ] please also make sure, The paragraph (e.g., "I'm a software ...") is a `p` tag.
      - This way will be semantically correct and the screen readers and other technologies will thank you
      
```
<h1> main title <h1>
<hx> sub title </hx>
<p> text <p>
```